common/source/docs/common-limited-firmware.rst

```diff
@@ -76,3 +76,19 @@ The missing features are listed below.
 
    -  Parachute
    -  Sprayer
+
+RAM Limitations
+===============
+
+Occasionally, in the above boards, RAM will become limited with enabled options in the firmware also. This is manifested by various symptoms:
```

Comment:
```suggestion
There may be insufficient RAM available in some flight controllers to support all enabled firmware features. This is manifested by various symptoms:
```

Reply:
Should probably be simply, `Some symptoms are`.

(the symptoms would be of the manifestation, but that's implied).

Reply:
done